libs/expo/shared/ui-components/src/lib/ImagePicker/index.tsx

```diff
@@ -57,32 +63,67 @@ export default function ImagePickerComponent(props: IImagePickerProps) {
         allowsEditing: false,
         allowsMultipleSelection: true,
       });
+
       if (!result.canceled && result.assets) {
-        const uploadPromises = result.assets.map(async (asset) => {
+        const newImages = result.assets.map((asset) => ({
+          id: undefined,
+          uri: asset.uri,
+          loading: true,
+          abortController: new AbortController(),
+        }));
+
+        setImages((prevImages) => {
+          if (prevImages) {
+            return [...prevImages, ...newImages];
+          }
+          return newImages;
+        });
+
+        const uploadPromises = newImages.map(async (image) => {
+          const asset = result.assets.find((a) => a.uri === image.uri);
+          if (!asset) return;
           const resizedPhoto = await resizeImage({ uri: asset.uri });
           const file = new ReactNativeFile({
             uri: resizedPhoto.uri,
             name: asset?.fileName || Date.now().toString(),
-            type: asset.mimeType || 'changeme',
+            type: asset?.mimeType || 'changeme',
```

Comment:
what's `'changeme'`?

Reply:
left that for me to find most usable and forgot. changed to imag jpeg